server/grpcservice/service.go

```diff
@@ -5,24 +5,32 @@ package grpcservice
 import (
 	"context"
 	"io"
+	"sync"
 	"time"
 
 	"github.com/CeresDB/ceresdbproto/pkg/metaservicepb"
 	"github.com/CeresDB/ceresmeta/pkg/log"
+	"github.com/CeresDB/ceresmeta/server/cluster"
+	"github.com/CeresDB/ceresmeta/server/member"
+	"github.com/pkg/errors"
 	"go.uber.org/zap"
 )
 
 type Service struct {
 	metaservicepb.UnimplementedCeresmetaRpcServiceServer
-
 	opTimeout time.Duration
 	h         Handler
+	manager   cluster.Manager
+
+	// Store as map[string]*grpc.ClientConn
+	grpcClientConns sync.Map
```

Comment:
- How about `connCache`?
- It seems there is no way to clear invalid connections and maybe we should take considerations for that.